app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -52,6 +54,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_trustees);
 
+        toolbar = (Toolbar)findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
         System.out.println(comradeEditText.size());
```

Comment:
@calistus
It wasn't me who wrote it actually. But yes, I'll make sure such lines are
not there next time.

On Thursday 14 July 2016, Ilo Calistus <notifications@github.com> wrote:

> In app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java
> <https://github.com/systers/PCSA-Android/pull/260#discussion_r70820481>:
>
> > @@ -52,6 +54,8 @@ protected void onCreate(Bundle savedInstanceState) {
> >          super.onCreate(savedInstanceState);
> >          setContentView(R.layout.activity_trustees);
> >
> > +        toolbar = (Toolbar)findViewById(R.id.toolbar);
> > +        setSupportActionBar(toolbar);
> >          System.out.println(comradeEditText.size());
>
> I hope you wrote this print statement when you were testing.
> Please remove it
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/systers/PCSA-Android/pull/260/files/af4accaa70ecde29981ebc065939f4a4c74aaef6#r70820481>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIiOkVlJ9-BMe1ER750khtHanpHHVfScks5qVk3cgaJpZM4JMSsi>
> .
>